simple/test.js

```diff
@@ -947,6 +954,13 @@ exports.test = function (global) {
     // edgeTests
     // /////////////////////////////////////////////////////////////////////////////
 
+    traversalProjections = function(params) {
+      // Note that depth 8 is good for all three sizes small (6), medium (7)
+      // and big (8). Depending on the size, we create a different tree.
+      let l = db._query(`FOR v IN 0..8 OUTBOUND "TreeV/S1:K1" GRAPH "Tree"
+                   RETURN v.data`, {}, {}, {}).toArray();
```

Comment:
We should definitely use the `{ silent }` option here, in the same way as the test directly below (`outbound`) does.
This is important, because the `silent` flag will drop the results on the coordinator / single server, so they will not be returned / converted to V8 objects. For queries with large result sets, we otherwise spend a lot of time in the results building / conversion, but here we are not interested in that.

Reply:
I had this at first, but I wasn't sure that this would then reproduce the timings I wanted to observe. Is it then not possible that the optimizer simply chooses not to fetch the data to the coordinator at all? And then my performance measurement does not make any sense any more...

Reply:
I think the only difference of `silent` here is that when the final step on coordinator / single server retrieves results from the root node Executor, it will discard them instead of putting them into a VPackBuilder / V8 array.
So I think it is safe to use `silent` here.

Reply:
ok